engine/src/main/java/org/terasology/engine/rendering/opengl/GLSLShader.java

```diff
@@ -364,7 +365,7 @@ private int compileShader(int type, Set<ShaderProgramFeature> features) {
     protected void doReload(ShaderData data) {
         try {
             GameThread.synch(() -> {
-                logger.debug("Recompiling shader {}.", getUrn());
+                logger.atDebug().addArgument(() -> getUrn()).log("Recompiling shader {}.");
```

Comment:
Simple getter method so impact neglectable
```suggestion
                logger.debug("Recompiling shader {}.", getUrn()); //NOPMD
```